### PR TITLE
PS-9044 Document new features in percona-202302 to percona-202305 (8.0.36.28)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/variables.md

### CODE REVIEW

File: docs/variables.md [hunk 1]
```diff
@@ -810,9 +865,13 @@ Maximum allowed value is `2000000` (two million delete markers).
 | Data type    | Boolean                                          |
 | Default      | OFF                                              |
```
Comment (patrickbirch) [inline docs/variables.md]: Shouldn't this be "ON"?

File: docs/variables.md [hunk 2]
```diff
@@ -810,9 +865,13 @@ Maximum allowed value is `2000000` (two million delete markers).
...
 
-Specifies whether to count single deletes as delete markers
-recognized by rocksdb_compaction_sequential_deletes.
-Disabled by default.
+!!! note
+
+    In version [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md) and later, the default value is changed from `OFF` to `ON`.
+
+The default value is `ON` which means the variable is enabled.
```
Comment (patrickbirch) [inline docs/variables.md]: Usual pattern is definition of the variable and then the default value.

File: docs/variables.md [hunk 3]
```diff
@@ -1484,8 +1559,14 @@ but also flushes all L0 files.
 | Data type    | Boolean                            |
 | Default      | OFF                                |
```
Comment (patrickbirch) [inline docs/variables.md:1560]: Should this be ON?

File: docs/variables.md [hunk 4]
```diff
@@ -1595,9 +1676,9 @@ Enabled by default.
 | Data type    | Boolean                |
 | Default      | TRUE                   |
 
-When enabled, this option allows index key prefixes longer than 767 bytes (up to
-3072 bytes). The values for rocksdb_large_prefix should be the same between
-source and replica.
+This variable is deprecated in `Percona Server for MySQL 8.0.36-28` and will be removed in a future release.
```
Comment (patrickbirch) [inline docs/variables.md:1680]: Do we have an alternative variable - what they should use instead of?

File: docs/variables.md [hunk 5]
```diff
@@ -1822,6 +1903,20 @@ The minimum value is `0` and the maximum value is `UINT64_MAX`.
...
+|--------------|-------------------------------------------------|
+| Command-line | --rocksdb-max-file-opening-threads              |
+| Dynamic      | No                                              |
+| Scope        | Global                                          |
+| Data type    | Numeric                                         |
+| Default      | 16                                              |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+This variable sets `DBOptions::max_file_opening_threads` for RocksDB. The default value is `16`. The minimum value is `1` and the maximum value is `INT_MAX`.
```
Comment (patrickbirch) [inline docs/variables.md]: INT_MAX?
The default type is Numeric, so we should have a number. 
INT_MAX = 9,223,372,036,854,775,807   (for 64-bit Integers)
Verify that this is the number.

File: docs/variables.md [hunk 6]
```diff
@@ -2074,6 +2169,22 @@ Specifies whether MyRocks should re-read the data file
...
+|--------------|---------------------------|
+| Command-line | --rocksdb-partial-index-ignore-killed |
+| Dynamic      | Yes                       |
+| Scope        | Global                    |
+| Data type    | Boolean                   |
+| Default      | ON                        |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+The dafault value is `ON` which means this variable is enabled.
```
Comment (patrickbirch) [inline docs/variables.md]: Should be definition of the variable and then the default value.

File: docs/variables.md (hunk 5 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: 32-біт (signed) = 2 147 483 647. Approved by Yura.

File: docs/variables.md (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: fixed

File: docs/variables.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/variables.md:1680]: We do not have an alternative variable. Approved by Yura.

File: docs/variables.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: fixed

File: docs/variables.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: fixed

**Review by patrickbirch — APPROVED** [review]
LGTM

File: docs/variables.md [hunk 7]
```diff
@@ -661,6 +684,20 @@ Turns on RocksDB memory-charging related features (BlockBasedTableOptions::cache
...
+|--------------|-------------------------|
+| Command-line | --rocksdb-check-iterate-bounds |
+| Dynamic      | Yes                     |
+| Scope        | Global, Session         |
+| Data type    | Boolean                 |
+| Default      | ON                      |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+This variable checks the upper and lower bounds of an iterator in RocksDB. The default value in `ON` which means this variable is enabled.
```
Comment (inikep) [inline docs/variables.md]: This variable enables checking of rocksdb iterator upper/lower bounds during iterating

File: docs/variables.md [hunk 8]
```diff
@@ -754,6 +791,20 @@ not.
...
+|--------------|----------------------|
+| Command-line | --rocksdb-compact-lzero-now |
+| Dynamic      | Yes                  |
+| Scope        | Global               |
+| Data type    | Boolean              |
+| Default      | OFF                  |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+This variable triggers a compaction of all `L0` files in RocksDB. The default value is `OFF`, which means the variable is disabled.
```
Comment (inikep) [inline docs/variables.md]: This variable is a switch. We call `rocksdb-compact-lzero-now=ON` to immediately compact L0 files and later it is automatically set to `OFF`.

File: docs/variables.md [hunk 9]
```diff
@@ -1484,8 +1562,14 @@ but also flushes all L0 files.
 | Data type    | Boolean                            |
 | Default      | OFF                                |
 
+!!! note
+
+    In version [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md) and later, the default value is changed from `ON` to `OFF`.
+
 Forces MyRocks to immediately flush all memtables out to data files.
 
+The default value is `OFF` which means the variable is disabled by default.
```
Comment (inikep) [inline docs/variables.md]: This variable is a switch. We call `rocksdb_force_flush_memtable_now=ON` to immediately flush all memtables and later it is automatically set to OFF.

**Review by inikep — CHANGES_REQUESTED** [review]

File: docs/variables.md (hunk 9 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: updated

File: docs/variables.md (hunk 8 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: updated

File: docs/variables.md (hunk 7 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: fixed

File: docs/variables.md [hunk 10]
```diff
@@ -754,6 +791,20 @@ not.
...
+|--------------|----------------------|
+| Command-line | --rocksdb-compact-lzero-now |
+| Dynamic      | Yes                  |
+| Scope        | Global               |
+| Data type    | Boolean              |
+| Default      | OFF                  |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+This variable acts as a switch. Set the variable to `ON`, `rocksdb-compact-lzero-now=ON`, to immediately compact all the `Level 0` (L0) files. After all the `L0` files are compacted, the variable value automatically switches to `OFF`. The default value is `OFF`, which means the variable is disabled.
```
Comment (inikep) [inline docs/variables.md]: This should be better: This variable acts as a trigger. 
Please remove: The default value is `OFF`, which means the variable is disabled.

File: docs/variables.md [hunk 11]
```diff
@@ -1484,7 +1562,11 @@ but also flushes all L0 files.
 | Data type    | Boolean                            |
 | Default      | OFF                                |
 
-Forces MyRocks to immediately flush all memtables out to data files.
+!!! note
+
+    In version [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md) and later, the default value is changed from `ON` to `OFF`.
+
+This variable acts as a switch. Set the variable to `ON`, `rocksdb_force_flush_memtable_now=ON`, to immediately flush all memtables. After all memtables are flushed, the variable value automatically switches to `OFF`. The default value is `OFF`, which means the variable is disabled.
```
Comment (inikep) [inline docs/variables.md]: This should be better: This variable acts as a trigger. 
Please remove: The default value is `OFF`, which means the variable is disabled.

**Review by inikep — APPROVED** [review]
LGTM after minor changes

File: docs/variables.md (hunk 10 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: fixed

File: docs/variables.md (hunk 11 above)
Comment (alina-derkach-oaza) [reply docs/variables.md]: fixed

**Pull request merged**